config.json

```diff
@@ -27,6 +27,16 @@
         "strings"
       ]
     },
+    { "slug": "resistor-color-duo",
+      "uuid": "57b0c57e-26a5-4ccf-aaf2-2fefddd918c1",
+      "core": true,
+      "unlocked_by": null,
```

Comment:
Shouldn't this be `"unlocked_by": "two-fer"`?

Reply:
No, cores are unlocked by null. It's the place in the file that defines the position in the track, for cores.  
(Why do even add this key for cores 🤔...? )

Reply:
Sorry, my bad!